.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Test colcon-ros-bundle
+on:
+  pull_request:
+  schedule:
+    - cron: '43 17 * * *'
+
+jobs:
+  run_test_suite:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.5
+    - name: Install dependencies
+      run: |
+        sudo apt update && sudo apt install -y enchant
+        pip3 install -U setuptools==44.0.0
+        pip3 install -r requirements_devel.txt
+        pip3 install -U git+https://github.com/colcon/colcon-bundle
+        pip3 install -U git+https://github.com/ros-infrastructure/rosdep
```

Comment:
That seems fragile - can't we use the release?

Reply:
This was in the `.travis.yml`. I'm assuming they wanted it this way to catch any breakage from the upstream deps.

Reply:
For colcon-bundle, this makes sense, `rosdep` is not maintained by us that's why it's more surprising. We should definitely install `colcon-bundle` from master. Maybe it'd be good to check why they need master from rosdep with Matt directly?

Reply:
> Well there’s two sets of tests. One that tests against mainline of all the deps and one that tests against the current pypi version. That’s so we can try to catch any bugs in mainline before they are released, specifically the colcon-* packages. - Matt